### ISSUE TITLE
Graphics issues on 4K monitor

### ISSUE THREAD

**Issue by readmodifywrite** [cryinkfly/Autodesk-Fusion-360-for-Linux#306]
For starters:
I did get this to work on a Thinkpad X1 laptop.  Awesome!

On my desktop:
Linux Mint 20
GPU: NVIDIA GP106 [GeForce GTX 1060 6GB]
Wine 8.0
Direct X 9 enabled

Fusion 360 will run, but the interface is too small to be usable.  The data panel works, but menus like file open do not - they just flash and then disappear.

Additionally, when exiting Fusion 360, around a dozen Windows processes are left open on the system.  I have to manually kill or kill -9 each of them.  (wineserver -k doesn't help, killall doesn't help, etc).  I'm  not sure if that is related or not (and it's something I can live with if it means running F360 natively!).

I have set the HiDPI mode on F360, it doesn't seem to have any effect.

If I try to change the Wine DPI setting, it does fix the UI scaling issues but the data panel won't load, making it impossible to open projects (since the menu dialogs like file open don't work either).

I have HiDPI set in Mint's settings: Mint really struggles without it on a 4K screen.  I did try various combinations of resolutions, fractional scaling, and disable HiDPI scaling, but nothing seems to help.

Any ideas or things I can try?

![2023-03-03_08-06](https://user-images.githubusercontent.com/3618507/222743501-3145e288-d536-4014-9bb5-fcbbd6f72442.png)

**Comment by readmodifywrite** [comment]
Here it is with Wine's DPI scaling turned up.  The grey space on the left is where the data panel should be.



![2023-03-03_08-21](https://user-images.githubusercontent.com/3618507/222744860-35a63267-f388-4a7c-957d-795cb32369f4.png)

**Comment by readmodifywrite** [comment]
Also tried with OpenGL - the UI is completely br

**Comment by readmodifywrite** [comment]
OK more notes:  I was on the Nvidia 470 driver, which is definitely not supported here.

I updated to the 525 driver.  It isn't perfect, but it's a huge improvement.  The file open dialog is working, and the data panel is functional (barely, but enough to scrape by).

If anyone has any tips to clean up the remaining glitches, I'm happy to try them out.  But I've got enough functionality working now to do what I need to do (and this is so much better than running in a VM I almost can't believe it!).  Feel free to close if you wish!


![2023-03-03_09-48](https://user-images.githubusercontent.com/3618507/222765006-969224ce-6a67-43f7-a66c-e2f04b357e82.png)
